api/android/api/src/main/jni/Android-snpe.mk

```diff
@@ -40,9 +40,8 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := snpe
 LOCAL_SRC_FILES := $(NNSTREAMER_FILTER_SNPE_SRCS)
-LOCAL_CXXFLAGS += -std=c++11 -frtti -fexceptions -Wno-exceptions -O2 -DNDEBUG $(NNS_API_FLAGS)
-LOCAL_C_INCLUDES := $(NNSTREAMER_INCLUDES) $(SNPE_INCLUDES) $(GST_HEADERS_COMMON)
-LOCAL_SHARED_LIBRARIES := libSNPE
-LOCAL_STATIC_LIBRARIES := nnstreamer
+LOCAL_CXXFLAGS := -std=c++11 -frtti -fexceptions -Wno-exceptions -O2 -DNDEBUG
+LOCAL_C_INCLUDES := $(SNPE_INCLUDES)
+LOCAL_SHARED_LIBRARIES := libSNPE $(NNS_API_LIBS)
 
 include $(BUILD_STATIC_LIBRARY)
```

Comment:
I verified with ` include $(BUILD_SHARED_LIBRARY) `.
You may change this line too!

Reply:
Updated. plz check it again.

Reply:
✔ Checked  ✔